### PR TITLE
Feat: dynamically set nav footer content by page url

### PR DESCRIPTION
**Pull request by tabathadelane**
Certain paths will now have their own cta's in the nav

These are the current directories with unique CTA's:

- /docs/new-relic-solutions/get-started
- /docs/new-relic-solutions/new-relic-one
- /docs/browser
- /docs/apm
- /docs/infrastructure
- /docs/logs
- /docs/mobile-monitoring
- /docs/network-performance-monitoring
- /docs/alerts-applied-intelligence
- /docs/query-your-data

### CODE REVIEW

File: src/components/NavFooter.js [hunk 1]
```diff
@@ -24,28 +41,62 @@ const NavFooter = ({ className }) => {
...
+        )}
+        {t(`navFooter.${ctaContent.i18nKey}`)}
         <Icon
           css={css`
             position: absolute;
             top: 4px;
+            right: -1em;
+            height: 1.1em;
+            width: 1.1em;
+            stoke: var(--erno-black);
```
Comment (LizBaker) [inline src/components/NavFooter.js:98]: lil typo 
```suggestion
            stroke: var(--erno-black);
```

File: src/components/NavFooter.js [hunk 2]
```diff
@@ -6,9 +6,26 @@ import {
...
+
+    for (const product of allCTAs) {
+      if (
+        currentPage.includes(ctaJson[product].directory) &&
+        product !== 'default'
+      ) {
+        ctaContent = ctaJson[product];
+      }
+    }
+  }
```
Comment (LizBaker) [inline src/components/NavFooter.js:27]: if you wanna remove the window check you could use the useLocation hook from reach/router here

Comment (tabathadelane) [reply src/components/NavFooter.js:27]: brilliant. thank you. i knew that didn't feel right.

**Review by LizBaker — APPROVED** [review]
simply beauteous

**Pull request merged**